<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TCPIP</class>
 <widget class="QWidget" name="TCPIP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>278</width>
    <height>353</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="tcpipBox">
     <property name="title">
      <string>TCP/IP</string>
     </property>
     <widget class="QLabel" name="localIpLabel">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>20</y>
        <width>201</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="label">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>20</y>
        <width>21</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ip</string>
      </property>
     </widget>
     <widget class="QTabWidget" name="tabWidget">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>60</y>
        <width>251</width>
        <height>171</height>
       </rect>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tabClient">
       <attribute name="title">
        <string>Client</string>
       </attribute>
       <widget class="QLabel" name="clientIpLabel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>21</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>IP</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="ipLineEdit">
        <property name="geometry">
         <rect>
          <x>50</x>
          <y>30</y>
          <width>141</width>
          <height>20</height>
         </rect>
        </property>
       </widget>
       <widget class="QLineEdit" name="clientPortLineEdit">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>59</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="clientPortLabel">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>50</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
       <widget class="QPushButton" name="connectButton">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Connect</string>
        </property>
       </widget>
       <widget class="QPushButton" name="clientStopButton">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>100</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tabServer">
       <attribute name="title">
        <string>Server</string>
       </attribute>
       <widget class="QLabel" name="serverPortLabel">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>40</y>
          <width>50</width>
          <height>21</height>
         </rect>
        </property>
        <property name="text">
         <string>Port</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="serverPortLineEdit">
        <property name="geometry">
         <rect>
          <x>70</x>
          <y>40</y>
          <width>91</width>
          <height>21</height>
         </rect>
        </property>
       </widget>
       <widget class="QPushButton" name="createButton">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>100</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Create</string>
        </property>
       </widget>
       <widget class="QPushButton" name="serverStopButton">
        <property name="geometry">
         <rect>
          <x>120</x>
          <y>100</y>
          <width>75</width>
          <height>23</height>
         </rect>
        </property>
        <property name="text">
         <string>Stop</string>
        </property>
       </widget>
      </widget>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
